rulesets/resources/301validator.xml

```diff
@@ -12,6 +12,8 @@
       <sch:assert role='ERROR' test='h:SoftwareInfo/h:extension/h:IECCERICalculation/h:Version[text()="2021" or text()="2018" or text()="2015"] or not(h:SoftwareInfo/h:extension/h:IECCERICalculation/h:Version)'>Expected SoftwareInfo/extension/IECCERICalculation/Version to be '2021' or '2018' or '2015'</sch:assert> <!-- See [IECC=2021] or [IECC=2018] or [IECC=2015] -->
       <sch:assert role='ERROR' test='count(h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version) &lt;= 1'>Expected 0 or 1 element(s) for xpath: SoftwareInfo/extension/EnergyStarCalculation/Version</sch:assert> <!-- Required if running ENERGY STAR calculation -->
       <sch:assert role='ERROR' test='h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version[text()="SF_National_3.0" or text()="SF_National_3.1" or text()="SF_National_3.2" or text()="SF_Pacific_3.0" or text()="SF_Florida_3.1" or text()="SF_OregonWashington_3.2" or text()="MF_National_1.0" or text()="MF_National_1.1" or text()="MF_National_1.2" or text()="MF_OregonWashington_1.2"] or not(h:SoftwareInfo/h:extension/h:EnergyStarCalculation/h:Version)'>Expected SoftwareInfo/extension/EnergyStarCalculation/Version to be 'SF_National_3.0' or 'SF_National_3.1' or 'SF_National_3.2' or 'SF_Pacific_3.0' or 'SF_Florida_3.1' or 'SF_OregonWashington_3.2' or 'MF_National_1.0' or 'MF_National_1.1' or 'MF_National_1.2' or 'MF_OregonWashington_1.2'</sch:assert> <!-- See [EnergyStar=SFNational30] or [EnergyStar=SFNational31] or [EnergyStar=SFNational32] or [EnergyStar=SFPacific30] or [EnergyStar=SFFlorida31] or [EnergyStar=SFOregonWashington32] or [EnergyStar=MFNational10] or [EnergyStar=MFNational11] or [EnergyStar=MFNational12] or [EnergyStar=MFOregonWashington12] -->
+      <sch:assert role='ERROR' test='count(h:SoftwareInfo/h:extension/h:ZERHCalculation/h:Version) &lt;= 1'>Expected 0 or 1 element(s) for xpath: SoftwareInfo/extension/ZERHCalculation/Version</sch:assert> <!-- Required if running ENERGY STAR calculation -->
+      <sch:assert role='ERROR' test='h:SoftwareInfo/h:extension/h:ZERHCalculation/h:Version[text()="Version_1"] or not(h:SoftwareInfo/h:extension/h:ZERHCalculation/h:Version)'>Expected SoftwareInfo/extension/ZERHCalculation/Version to be 'Version_1'</sch:assert> <!-- See [ZERH=Version_1] -->
```

Comment:
Seems redundant that Version includes "Version" in the name. Can we change all files to use "1.0" instead of "Version_1"?